build/compile-validators.js

```diff
@@ -60,7 +62,9 @@ const schemas = {
   ProtocolsQuery,
   RecordsRead,
   PublicJwk,
-  SnapshotsCreate
+  SnapshotsCreate,
+  BaseDecodedAuthorizationPayload,
+  RecordsWriteDecodedAuthorizationPayload
```

Comment:
Consider leaving `Decoded` out: we can think of payload being either "as is" or "encoded".

```suggestion
  BaseAuthorizationPayload,
  RecordsWriteAuthorizationPayload
```

Reply:
Updated